### PR TITLE
[3372] Lead schools list

### PR DESCRIPTION
**Pull request by mooktakim**
### Context

https://trello.com/c/6Nk2hZD5/3372-s-lead-schools-list

### Changes proposed in this pull request

* New section in system admin for lead schools

### Guidance to review

* login as admin user and go to system admin section, click on lead schools

### Important business

* Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
* Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?

### CODE REVIEW

**Comment by mooktakim** [comment]
[[S] Lead schools list](https://trello.com/c/6Nk2hZD5/3372-s-lead-schools-list)

File: config/locales/en.yml [hunk 1]
```diff
@@ -346,6 +346,8 @@ en:
         delete: Remove a user
       validation_errors:
         page_title: Validation errors
+      lead_schools:
+        index: Lead Schools
```
Comment (AbigailMcP) [inline config/locales/en.yml:350]:
```suggestion
        index: Lead schools
```

**Review by AbigailMcP — APPROVED** [review]
It's beautiful ⭐

**Comment by bbelward** [comment]
tiny change - could we update the name column heading to say Lead school name? Good to be more specific in case we want to export in future. The schools list also has 'School name' so would be good to match up for consistency, so Lead schools page says 'Lead school name' and Schools list says 'School name' 
Screenshot 2021-12-24 at 10 54 24
Screenshot 2021-12-24 at 10 54 08

**Pull request merged**